modules/cjson/Kconfig

```diff
@@ -4,13 +4,8 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-# cJSON Library for Zephyr
-
-config  CJSON_LIB
+config CJSON_LIB
 	bool "cJSON library"
 	# Since minimal libc lacks sscanf and strtod we need to use newlibc
 	# Allow external libc to allow native posix to be used for unit test
 	depends on NEWLIB_LIBC || EXTERNAL_LIBC
-	default n
-	help
-	  Enable the cJSON Library
```

Comment:
Frankly, I didn't view it as really helpful. IMO its quite obvious that this configuration is to enable cJSON. Since most configs enable things.

Reply:
Fair enough.

Reply:
Just be aware that the help text is rendered into documentation for nRF Connect SDK:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/CONFIG_CJSON_LIB.html?highlight=config_cjson_lib#cmdoption-arg-CONFIG_CJSON_LIB

So just because of that a help text is always nice.